addons/mail/models/discuss/ir_websocket.py

```diff
@@ -22,10 +24,16 @@ def _get_im_status(self, data):
     @add_guest_to_context
     def _build_bus_channel_list(self, channels):
         channels = list(channels)  # do not alter original list
+        channel_ids = list()
+        for channel in channels:
+            if isinstance(channel, str):
+                match = re.findall(r'discuss\.channel.*?(\b\d+\b)', channel)
```

Comment:
Why is the regex so complex? Cant' we just send a simple format without extra `\b` and `.*?`?
`discuss\.channel_(\d+)` ?